packages/actor-init-query/test/ActorInitQuery-test.ts

```diff
@@ -772,6 +772,22 @@ LIMIT 100
         });
       });
 
+      it('handles the --cache flag', async() => {
+        const stdout = await stringifyStream(<any> (await actor.run({
+          argv: [ sourceHypermedia, '-q', queryString, '--cache', 'false' ],
```

Comment:
I guess this should be `--noCache`?

Reply:
Ok, changed the argument to `--noCache` instead of `--cache false`